### ISSUE TITLE
Icons not (always) loaded with default templates

### ISSUE THREAD

**Issue by YannickRe** [microsoft-search/pnp-modern-search#171]
**Version used**
3.11.0

**Describe the bug**
The FileIcon is not always loaded on the Search Results web part. Chrome Dev Console shows
```
The icon "spo16_svg" was used but not registered. See http://aka.ms/fabric-icon-usage for more information.
```

**To Reproduce**
1. Put a Search Results web part on the page
2. Make it search for `contentclass:STS_Site` (make sure you have a classic site, or modern non-Group connected site in the results on the first page)
3. Select the DetailsList template
4. Save the page
5. Do a full reload

**Expected behavior**
The icon shows up in the list.

**Screenshots**
![Capture](https://user-images.githubusercontent.com/9973962/78377429-e3474400-75cf-11ea-88a3-3587b1eef289.PNG)

**Desktop (please complete the following information):**
 - Browser Edgium/CrEdge

**Additional context**
Problem comes from these lines of code:
https://github.com/microsoft-search/pnp-modern-search/blob/039f796121471a1b9d192e0b9cafe42d606ddc60/search-parts/src/services/TemplateService/BaseTemplateService.tsx#L547-L550
Since DetailsList is moved to a web component which in turn uses the Icon web component, the `indexOf` doesn't find a reference to the usage of the icon.

I'd fix it myself but I am a bit out of tune with the current state, so in this particular case I have no idea how to consistently check whether or not an icon has been used in the template.

**Comment by wobba** [comment]
Thank you for reporting and we'll sort it out. Adding the loading to the different web components using it could be a solution.

**Comment by wobba** [comment]
Fixed with #171
